builtin/providers/aws/resource_aws_cloudwatch_event_rule.go

```diff
@@ -218,7 +222,7 @@ func buildPutRuleInputStruct(d *schema.ResourceData) *events.PutRuleInput {
 		input.Description = aws.String(v.(string))
 	}
 	if v, ok := d.GetOk("event_pattern"); ok {
-		pattern, _ := normalizeJsonString(v.(string))
+		pattern, _ := normalizeJsonString(v)
```

Comment:
We can change the signature without too much of an issue, it might just complicate the call site a bit. Happy to leave this for a future PR though since it's strictly better than what is there now.